commerce_coordinator/apps/frontend_app_payment/pipeline.py

```diff
@@ -0,0 +1,25 @@
+"""
+Legacy filter pipeline implementation
+"""
+
+from django.conf import settings
+from openedx_filters import PipelineStep
+
+from commerce_coordinator.apps.frontend_app_payment.constants import FRONTEND_APP_PAYMENT_CHECKOUT
+
+
+class GetLegacyRedirectUrl(PipelineStep):
+    """
+    Generates legacy redirect url
+    """
+    def run_filter(self, active_order_management_system, redirect_url):  # pylint: disable=arguments-differ
```

Comment:
Can you confirm if the return property from 1st pipeline step (i.e. `redirect_url`) is propagated here to this (2nd) pipeline step as a method argument?

Reply:
I can confirm that it is.